v4l2loopback.c

```diff
@@ -2187,17 +2187,30 @@ static int allocate_buffers(struct v4l2_loopback_device *dev)
 
 	dprintk("allocating %ld = %ldx%d\n", dev->imagesize, dev->buffer_size,
 		dev->buffers_number);
+	int err = -ENOMEM;
```

Comment:
ISO C90 forbids mixed declarations and code

Reply:
Fixed.